cedar-lean/CedarFFI/Main.lean

```diff
@@ -800,4 +803,59 @@ private def ignoreOutput (vc : SymEnv → Cedar.SymCC.Result Cedar.SymCC.Asserts
     let data := (String.fromUTF8? inner_buffer.data).getD ""
     return ({ data := data, duration := r.duration } : Timed String)
 
+def EntityLoader.mk (entities: Cedar.Spec.Entities) : EntityLoader :=
```

Comment:
I think the lean codebase already has a test entity loader constructor that does the same thing.